src/router/index.ts

```diff
@@ -39,6 +39,8 @@ router.get('/products/type', products.routes.productsTypeGet);
 
 router.post('/orders', orders.validation.ordersPost, orders.routes.ordersPost);
 
+router.post('/orders/glovo', orders.routes.glovoOrdersPost);
```

Comment:
please use endpoint from the [official documentation](https://api-docs.glovoapp.com/partners/index.html#section/Getting-started/Event-notifications-using-webhooks)